src/test/java/org/jabref/logic/importer/fetcher/ArXivTest.java

```diff
@@ -33,18 +38,17 @@ void setUp() {
         when(importFormatPreferences.getKeywordSeparator()).thenReturn(',');
         finder = new ArXiv(importFormatPreferences);
         entry = new BibEntry();
-
-        sliceTheoremPaper = new BibEntry();
-        sliceTheoremPaper.setType(StandardEntryType.Article);
-        sliceTheoremPaper.setField(StandardField.AUTHOR, "Tobias Diez");
-        sliceTheoremPaper.setField(StandardField.TITLE, "Slice theorem for Fréchet group actions and covariant symplectic field theory");
-        sliceTheoremPaper.setField(StandardField.DATE, "2014-05-09");
-        sliceTheoremPaper.setField(StandardField.ABSTRACT, "A general slice theorem for the action of a Fr\\'echet Lie group on a Fr\\'echet manifolds is established. The Nash-Moser theorem provides the fundamental tool to generalize the result of Palais to this infinite-dimensional setting. The presented slice theorem is illustrated by its application to gauge theories: the action of the gauge transformation group admits smooth slices at every point and thus the gauge orbit space is stratified by Fr\\'echet manifolds. Furthermore, a covariant and symplectic formulation of classical field theory is proposed and extensively discussed. At the root of this novel framework is the incorporation of field degrees of freedom F and spacetime M into the product manifold F * M. The induced bigrading of differential forms is used in order to carry over the usual symplectic theory to this new setting. The examples of the Klein-Gordon field and general Yang-Mills theory illustrate that the presented approach conveniently handles the occurring symmetries.");
-        sliceTheoremPaper.setField(StandardField.EPRINT, "1405.2249");
-        sliceTheoremPaper.setField(StandardField.FILE, ":http\\://arxiv.org/pdf/1405.2249v1:PDF");
-        sliceTheoremPaper.setField(StandardField.EPRINTTYPE, "arXiv");
-        sliceTheoremPaper.setField(StandardField.EPRINTCLASS, "math-ph");
-        sliceTheoremPaper.setField(StandardField.KEYWORDS, "math-ph, math.DG, math.MP, math.SG, 58B99, 58Z05, 58B25, 22E65, 58D19, 53D20, 53D42");
+        sliceTheoremPaper = new BibEntry(StandardEntryType.Article);
+
+        sliceTheoremPaper.withField(StandardField.AUTHOR, "Tobias Diez")
```

Comment:
```suggestion
        .withField(StandardField.AUTHOR, "Tobias Diez")
```

Just chain it with the last statement.